pyproject.toml

```diff
@@ -23,6 +24,9 @@ pre-commit = "^3.7.0"
 isort = "^5.13.1"
 pyright = "^1.1.362"
 pytest-nhsd-apim = "^3.3.12"
+cryptography = "^42.0.5"
+python-dateutil = "^2.9.0.post0"
```

Comment:
Can't see where python-dateutil is being used.

Reply:
It was at some desperate point. Good spot.